eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -132,7 +132,6 @@ public function load(array $configs, ContainerBuilder $container)
             throw new InvalidArgumentException($message);
         }
 
-        $this->handleSiteAccessesRelation($container);
```

Comment:
> removed Site Access relation handling (which is replaced by a dedicated method in SiteAccessService in the previous PR).

Does this mean that the parameter `ezpublish.siteaccess.relation_map` is entirely gone and a developer needs to upgrade his code to use the new approach? If so, a proper bc note explaining that is due.